MeetHut.Services/Meet/RoomService.cs

```diff
@@ -0,0 +1,35 @@
+using AutoMapper;
+using MeetHut.DataAccess;
+using MeetHut.DataAccess.Entities.Meet;
+using MeetHut.Services.Meet.DTOs;
+using MeetHut.Services.Meet.Models;
+using Microsoft.Extensions.Configuration;
+using Livekit.Client;
+
+namespace MeetHut.Services.Meet
+{
+    /// <inheritdoc cref="MeetHut.Services.Meet.IRoomService" />
+    public class RoomService : MapperRepository<Room, RoomDTO, RoomModel>, IRoomService
+    {
+        /// <summary>
+        /// Init Room Service
+        /// </summary>
+        /// <param name="databaseContext">Database Context</param>
+        /// <param name="mapper">Mapper</param>
+        /// <param name="configuration">Configuration</param>
+        public RoomService(DatabaseContext databaseContext, IMapper mapper, IConfiguration configuration) : base(databaseContext, mapper)
+        {
+            System.Console.WriteLine("Just testing Livekit connection...");
```

Comment:
I think it should be in a function.

Reply:
It's only until we really create the room service for the app. Then it will be deleted/moved.

Reply:
I can accept it.